Makefile

```diff
@@ -271,4 +271,24 @@ dockerized-build-amd64:
 
 # Use vendor instead of proxy when building w/ vendor folder
 dockerized-build-vendor:
-	$(DOCKER_BUILD_FROM_SOURCE) --build-arg GO111MODULE=off .
\ No newline at end of file
+	$(DOCKER_BUILD_FROM_SOURCE) --build-arg GO111MODULE=off .
+
+.PHONY: create-eks-cluster
+create-eks-cluster:
+	@eksctl create cluster \
+		--name $(namespace) \
+		--region us-east-1 \
```

Comment:
is it safe to assume that we're always deploying to us-east-1?

Reply:
for our use cases, yes it is safe. i'll fix it up and add region as a parameter just in case